components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -2059,7 +2059,8 @@ private OMElement queryToGetAPIUsageByResourcePath(String columnFamily, String f
         }
     }
 
-    private OMElement queryToGetAPIUsageByDestination(String columnFamily, String fromDate, String toDate)
+    private List<APIUsageByDestination> queryToGetAPIUsageByDestination(String tableName, String fromDate,
+            String toDate)
```

Comment:
rename method as "getAPIUsageByDestinationData" and fixed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621